make/deps.mk

```diff
@@ -77,7 +77,7 @@ dep_erlcloud = git https://github.com/erlcloud/erlcloud 3.2.4
 
 dep_erlazure = git https://github.com/lazedo/erlazure.git add-start-link
 
-dep_couchbeam = git https://github.com/2600hz/erlang-couchbeam 86d946646ad9dee1dee305b15d5c59e4c8e60cc2 # as per kazoo master branch commit SHA refered from 2600hz-kazoo5 branch
+dep_couchbeam = git https://github.com/vonix-networks/erlang-couchbeam 1.5-vonix # Created new branch branch from kazoo5 and merged master into it
```

Comment:
As discussed in my message please use 1.5.0-vonix (the tag) not the tracking branch 1.5-vonix